lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -578,6 +606,78 @@ def test_score_without_student_state(self):
         self.assertEquals(student_module.grade, 0.75)
         self.assertEquals(student_module.max_grade, 1)
 
+    @XBlock.register_temp_plugin(StubCompletableXBlock, identifier='comp')
+    def test_completion_event(self):
```

Comment:
Consider having a test for waffle disabled for these first two tests.  It is somewhat low risk given that you are protected by raising the error at this point, so not a big deal.  It just means if a bug skips in you may find it a little later.